### PR TITLE
[BACK-54] Second Version of dbUpdater

### PR DESCRIPTION
**Pull request by thomas-woehrle**
### Description of Issue
Improvement of the first version. 

This includes:
- distinction between users or organizations as owners
- integration of star-history and languages used in a project
- integration of updateRepos function
- adjusting the db correspondingly
### How I implemented
- restructured the calling of functions
- added` is_trending_daily, is_trending_weekly, is_trending_monthly` on db 
- these columns are purged and updated daily
- also added ELI5 integration of Maxis and Alex' functions
### Other
- right now updateProject and insertProject do very similar things but later on hackernewsPosts for example will not be updated on a daily basis

### CODE REVIEW

File: packages/graphql_server/package.json
Comment (pczern) [inline packages/graphql_server/package.json:18]: if this dependency is added here, then remove the supabase dependency, delete package-lock.json/node_modules and run npm i again

Comment (pczern) [reply packages/graphql_server/package.json:18]: we will use supabase-js dependency to fix possible former deployment bug

File: packages/graphql_server/package.json
Comment (pczern) [reply packages/graphql_server/package.json:15]: get project-id from env variables

File: packages/graphql_server/src/dataAggregation.ts
Comment (pczern) [inline packages/graphql_server/src/dataAggregation.ts:203]: ternary expression

**Review by pczern — CHANGES_REQUESTED** [review]

File: packages/graphql_server/src/processRepo.ts
Comment (thomas-woehrle) [reply packages/graphql_server/src/processRepo.ts:73]: Would prefer to keep. We will move to a logger later on, but for now it is nice to see what is happening when running this function.

File: packages/graphql_server/types/dataAggregation.d.ts
Comment (thomas-woehrle) [reply packages/graphql_server/types/dataAggregation.d.ts:5]: When i do it this way and hover over a type of ProjectInsertion i get an odd description of what the type looks like. So would prefer to keep like this.

**Comment by thomas-woehrle** [comment]
@pczern @cnk1dev 
I addressed your change requests

File: packages/graphql_server/src/dataAggregation.ts
Comment (cnk1dev) [inline packages/graphql_server/src/dataAggregation.ts:212]: Did you mean to use person instead of existing_person here?

**Review by cnk1dev — APPROVED** [review]
Double-check the return statement in the comment below, otherwise good to go from my side!

File: packages/graphql_server/types/dataAggregation.d.ts
Comment (pczern) [reply packages/graphql_server/types/dataAggregation.d.ts:5]: if there is no error, change it please; you are not required to hover

File: packages/graphql_server/package.json
Comment (pczern) [reply packages/graphql_server/package.json:15]: if you us `$npm_package_config_env_dev && <your command>` and then get the environment variable like $SUPABASE_REFERENCE_ID it will work

Comment (thomas-woehrle) [reply packages/graphql_server/package.json]: thanks, changed it

**Comment by thomas-woehrle** [comment]
I will not merge until BACK-56 is merged to check for conflicts

**Comment by thomas-woehrle** [comment]
Never mind might be better to merge this one first

**Pull request merged**
